frontend/public/components/storage-class.tsx

```diff
@@ -18,23 +18,19 @@ const StorageClassHeader = props => <ListHeader>
   <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
   <ColHead {...props} className="col-sm-4 col-xs-6" sortField="provisioner">Provisioner</ColHead>
   <ColHead {...props} className="col-sm-2 hidden-xs" sortField="reclaimPolicy">Reclaim <span className="hidden-sm">Policy</span></ColHead>
-  <ColHead {...props} className="col-sm-2 hidden-xs" sortField={`metadata.annotations['${defaultClassAnnotation}']`}>Default <span className="hidden-sm">Class</span></ColHead>
```

Comment:
+1 to @jhadvig's comment. Columns no longer add up to 12.

Reply:
Is it fine if I change it to 6-4-2? Giving 6 to 'Name' and so on.

Reply:
5-5-2 maybe?

Reply:
Okay :) Ack.